network/message/message.proto

```diff
@@ -8,9 +8,6 @@ package message;
 // gossip network
```

Comment:
nit: whitespace in:
```
 syntax = "proto3";
```

You should likely format the protofile with `buf` (buf.build)

Reply:
done